modules/manage/pages/iceberg/topic-iceberg-integration.adoc

```diff
@@ -114,15 +134,15 @@ rpk registry schema create <subject-name> --schema </path-to-schema> --type <for
 +
 [,bash,role=no-copy]
 ----
-SUBJECT                VERSION   ID   TYPE
+SUBJECT          VERSION   ID   TYPE
 <subject-name>   1         1    PROTOBUF
 ----
 
-The Iceberg table is inside a namespace called `redpanda`, and has the same name as the Redpanda topic name. As you produce records to the topic, the data also becomes available in object storage for consumption by Iceberg-compatible clients. 
+The Iceberg table is inside a namespace called `redpanda`, and has the same name as the Redpanda topic name. As you produce records to the topic, the data also becomes available in object storage for consumption by Iceberg-compatible clients. You can use the same analytical tools to xref:manage:iceberg/query-iceberg-topics.adoc[read the Iceberg topic data] in a data lake as you would for a relational database.
 
-== Schema support and mapping
+== About schema support and translation to Iceberg format
 
-The `redpanda.iceberg.mode` property determines how Redpanda maps the topic data to the Iceberg table structure. You can either have the generated Iceberg table match the stucture of a Avro or Protobuf schema in the Schema Registry, or use the `key_value` mode where Redpanda stores the record values as-is in the table. 
+The xref:reference:properties/topic-properties.adoc#redpanda-iceberg-mode[`redpanda.iceberg.mode`] property determines how Redpanda maps the topic data to the Iceberg table structure. You can either have the generated Iceberg table match the structure of a Avro or Protobuf schema in the Schema Registry, or use the `key_value` mode where Redpanda stores the record values as-is in the table. 
 
 The JSON Schema format is not supported. If your topic data is in JSON, it is recommended to use the `key_value` mode.
```

Comment:
```suggestion
The JSON schema format is not supported. If your topic data is in JSON, it is recommended to use the `key_value` mode.
```

Reply:
This is kind of a weird one because the official name of the format is "JSON Schema," but when it is part of the group of formats (Avro, Protobuf, JSON Schema) we use lowercase and let "schemas" refer to the collective. I'll keep the capitalization since we already use "JSON Schema" in our other docs.